sdccc/src/main/java/com/draeger/medical/sdccc/manipulation/precondition/impl/ManipulationPreconditions.java

```diff
@@ -398,6 +408,29 @@ static Optional<String> associateNewLocationForHandle(
             return stateHandle;
         }
 
+        private static void waitForEpisodicContextReportToModifyStateHandle(
+                final TestClient testClient, final String stateHandle) {
+            final EpisodicContextReportStateHandleObserver observer =
+                    new EpisodicContextReportStateHandleObserver(stateHandle);
+            testClient.getSdcRemoteDevice().getMdibAccessObservable().registerObserver(observer);
+
+            final long start = System.currentTimeMillis();
```

Comment:
Not really needed, you could use _now_ variable to determine the value for _end_

Reply:
Done.